network/policy/policy_windows.go

```diff
@@ -353,10 +353,12 @@ func GetHcnEndpointPolicies(policyType CNIPolicyType, policies []Policy, epInfoD
 		if policy.Type == policyType {
 			var err error
 			var endpointPolicy hcn.EndpointPolicy
+			var isOutbountNatPolicy bool
 
 			switch GetPolicyType(policy) {
 			case OutBoundNatPolicy:
 				endpointPolicy, err = GetHcnOutBoundNATPolicy(policy, epInfoData)
+				isOutbountNatPolicy = true
```

Comment:
it looks like there is an issue SNAT config file also. Could you look into that also.

Reply:
what's the issue or context?